features/step_definitions/basic_steps.rb

```diff
@@ -1,7 +1,41 @@
 When(/^I visit "([^"]*)" page$/) do |page_name|
-  visit root_path
+
+  page_path = get_path page_name
```

Comment:
Really no need for the extra variable here IMO, you can just call the method which will return the value to the `visit` method call